src/lib/openapi/spec/feature-dependency-schema.ts

```diff
@@ -0,0 +1,35 @@
+import { FromSchema } from 'json-schema-to-ts';
+import { createDependentFeatureSchema } from './create-dependent-feature-schema';
+import { dependentFeatureSchema } from './dependent-feature-schema';
+
+export const featureDependencySchema = {
+    $id: '#/components/schemas/featureDependencySchema',
+    type: 'object',
+    description:
+        'Feature dependency connection between a child feature and its parents',
+    required: ['child', 'parents'],
+    additionalProperties: false,
+    properties: {
+        child: {
```

Comment:
we don't use child and parent in the openapi. We use feature and dependencies instead. I'd stick to consistent naming.